pages/linux/rename.md

```diff
@@ -1,6 +1,7 @@
 # rename
 
 > Renames multiple files.
```

Comment:
```suggestion
> Rename multiple files.
```
While we're at it, let's also correct this and make it impersonal (really strange that the man page uses third person). Same goes for `rename.ul`.

Reply:
Fixed, thanks!